rosbridge_library/src/rosbridge_library/capabilities/service_response.py

```diff
@@ -34,6 +34,7 @@ def service_response(self, message):
             message_conversion.populate_instance(values, resp)
             # pass along the response
             service_handler.handle_response(request_id, resp)
+            self.protocol.send(message)
```

Comment:
wasn't sure if this was needed, but I needed it to reproduce the old tests' behavior -- but maybe the tests are outdated